### PR TITLE
fix: IM-4143-bring reference line in front of bars in distribution plot chart

### PR DESCRIPTION
**Pull request by sharleneaqlik**
**The issue**: The Reference line on the distributionplot chart is always behind the “bars” of the chart. 

Screenshot reference line

Fix: The reference line now should be in front of the bars.
 
image

### CODE REVIEW

File: chart-modules/common/picasso/chart-builder/presets/dimension-measure-chart/layout-settings.js [hunk 1]
```diff
@@ -20,9 +20,11 @@ export default {
   },
 
   refLines: {
-    minimumLayoutMode: 'SPARK',
-    prioOrder: 60,
-    displayOrder: -10,
+    layout: {
+      minimumLayoutMode: 'SPARK',
+      prioOrder: 60,
+      displayOrder: 10000,
```
Comment (T-Wizard) [inline chart-modules/common/picasso/chart-builder/presets/dimension-measure-chart/layout-settings.js:26]: would a slightly lower value work?
I would like it to be smaller then the displayOrder for selections components (ex: lasso + range select)

Comment (sharleneaqlik) [reply chart-modules/common/picasso/chart-builder/presets/dimension-measure-chart/layout-settings.js:26]: To be honest I don't know what value range is supposed to be used. So what number do you suggest?

Comment (T-Wizard) [reply chart-modules/common/picasso/chart-builder/presets/dimension-measure-chart/layout-settings.js:26]: I recommend a new number (not used by anything else)
smaller then 10000 and larger then anything that it should render over (probably >1100 if it should render over points in  dist-plot)

Comment (sharleneaqlik) [reply chart-modules/common/picasso/chart-builder/presets/dimension-measure-chart/layout-settings.js:26]: Ok, will try and update.

**Review by T-Wizard — APPROVED** [review]

**Pull request merged**